tests/content/src/main/content/jcr_root/content/typerefinery-showcase/pages/content/footer/.content.xml

```diff
@@ -29,25 +29,12 @@
-            <label jcr:primaryType="nt:unstructured" sling:resourceType="typerefinery/components/forms/fields/label" label="Enter Email Address" hideLabel="true">
-            </label>
-            <field jcr:primaryType="nt:unstructured" sling:resourceType="typerefinery/components/forms/fields/input" inputType="email" placeholder="Eg. test@example.com"/>
-          </input_1>
-          <button_1 jcr:primaryType="nt:unstructured" sling:resourceType="typerefinery/components/forms/fields/button" label="Submit" lgColSize="2">
-          </button_1>
-        </container4_1>
-      </container4>
-      <container5 jcr:primaryType="nt:unstructured" sling:resourceType="/apps/typerefinery/components/layout/container" lgColSize="12" mdColSize="12" smColSize="12" flexEnabled="true" horizontalAlignment="between" classes="p-2" borderPositionTop="true">
+      <container5 jcr:primaryType="nt:unstructured" sling:resourceType="/apps/typerefinery/components/layout/container" lgColSize="12" mdColSize="12" smColSize="12" flexEnabled="true" horizontalAlignment="between" classes="p-2 mt-4" borderPositionTop="true">
         <container1 jcr:primaryType="nt:unstructured" sling:resourceType="/apps/typerefinery/components/layout/container" lgColSize="8" mdColSize="12" smColSize="12" flexEnabled="true">
           <title jcr:primaryType="nt:unstructured" sling:resourceType="typerefinery/components/content/title" title ="Copyright © 2023 Typerefinery. /All Rights Reserved. /Privacy Terms of Use" headingLevel="h6"/>
         </container1>
         <container2 jcr:primaryType="nt:unstructured" sling:resourceType="typerefinery/components/layout/container" lgColSize="4" mdColSize="12" smColSize="12" columnGap="3" flexEnabled="true" horizontalAlignment="end">
-          <button_1 jcr:primaryType="nt:unstructured" sling:resourceType="typerefinery/components/forms/fields/button" buttonType="navigate" buttonVariant="info" isOutlinedButton="true" label="Github" showIcon="true" icon="pi pi-github" lgColSize="4" mdColSize="6" navigateTo="https://github.com/typerefinery-ai/typerefinery-websight" openInNewTab="true" classes="btn-sm"></button_1>
-          <button_2 jcr:primaryType="nt:unstructured" sling:resourceType="typerefinery/components/forms/fields/button" buttonType="navigate" buttonVariant="info" isOutlinedButton="true" label="About Us" showIcon="true" icon="pi pi-globe" lgColSize="4" mdColSize="6" navigateTo="https://typerefinery.ai/" openInNewTab="true" classes="btn-sm"></button_2>
+          <button_1 jcr:primaryType="nt:unstructured" sling:resourceType="typerefinery/components/forms/fields/button" buttonType="navigate" buttonVariant="primary" isOutlinedButton="true" label="Github" showIcon="true" icon="pi pi-github" lgColSize="4" mdColSize="6" navigateTo="https://github.com/typerefinery-ai/typerefinery-websight" openInNewTab="true" classes="btn-sm"></button_1>
```

Comment:
I think in this PR you have changed button variant to button style.

then why in some places you are using button variant and in some places you are using button style?